app/Dockerfile

```diff
@@ -6,13 +6,13 @@
 # After running `make release-build` (see [Releases](../docs/releases.md)),
 # you can verify the image works by running
 #
-#   docker run -p 80:8000 template-infra-app
+#   docker run -e PORT=8000 -p 80:8000 template-infra-app
 #
 # then visit localhost in a browser (or can use curl from a terminal)
 
 FROM busybox:uclibc AS release
 
-ARG RUN_USER=nodummy
+ARG RUN_USER=nobody
```

Comment:
It looks like we missed this bug in #203 since the "Template CI Infra" workflow didn't run in that PR, so could we add `app/Dockerfile` or `app/**` to the `paths` in `template-only-ci-infra.yml`

Reply:
Starting with `app/Dockerfile` for now, we can swap it to be the dual wildcards down the road if we see this become another issue